install.sh

```diff
@@ -15,6 +15,18 @@ NC='\033[0m' # No Color
 printf '%bsuiup installer script%b\n' "${CYAN}" "${NC}"
 printf 'This script will install the suiup binary to your system.\n'
 
+# Environment variables for controlling behavior:
+# SUIUP_INSTALL_DIR - Custom installation directory
+# SUIUP_SKIP_CHECKSUM - Set to "true" to skip file integrity verification (NOT RECOMMENDED)
+# GITHUB_TOKEN - GitHub token for API authentication (avoids rate limits)
+#
+# Security Note: Skipping integrity verification reduces security. Only use SUIUP_SKIP_CHECKSUM 
+# in trusted environments or when you can manually verify the downloaded binary.
+
+# Global flag to track if integrity verification was skipped
+INTEGRITY_CHECK_SKIPPED=false
```

Comment:
nit: I wonder if it makes sense to rename this
```suggestion
SKIP_INTEGRITY_CHECK=false
```